### ISSUE TITLE
misc/cgo/test: TestParallelSleep is still flaky

### ISSUE THREAD

**Issue by bcmills** [golang/go#45586]
[2021-04-15T12:38:13-7ad496b/windows-amd64-2012](https://build.golang.org/log/7b94fd917ca7cd4fa885d837c142b559bb5b2926)
[2021-04-14T00:25:33-8dcc071/linux-ppc64le-buildlet](https://build.golang.org/log/dd476468cfb91c326b68eb0c66b795f887abff01)
[2020-04-19T03:12:32-a32262d/netbsd-amd64-9_0](https://build.golang.org/log/43f01b36c1651d7b2d6df0809815fcbc6723fc3d)

```
--- FAIL: TestParallelSleep (1.57s)
    testx.go:225: difference in start time for two sleep(1) is 562ms
    testx.go:229: parallel 1-second sleeps slept for 0.562000 seconds
FAIL
```

See previously #6636, #5638, #5480, #4008, #3964.

CC @ianlancetaylor

**Comment by ianlancetaylor** [comment]
Very odd that this would have suddenly occurred on three different platforms.  Assuming that the original bug was not recurred, this is basically testing how long it takes to start a new goroutine.  Very hard to believe that it would take half a second when nothing else should be going on.

CC @mknyszek @prattmic

**Comment by bcmills** [comment]
`greplogs --dashboard -md -l -e 'FAIL: TestParallelSleep' --since=2021-04-16`

[2021-11-04T07:05:31-2622235/windows-amd64-2012](https://build.golang.org/log/a741ffa4d911c55c46feacc0a29b23d270d277e3)
[2021-08-26T17:18:58-d6bdae3/windows-amd64-noregabi](https://build.golang.org/log/b64adefd6fa38cca358544897dd3c44c4016acad)
[2021-08-20T03:25:17-c92c2c9/windows-amd64-2016](https://build.golang.org/log/e9b699b522bb1707db46c8b1e5e4c7ca0596d977)
[2021-05-27T19:55:46-cdcd028/openbsd-arm64-jsing](https://build.golang.org/log/7cea9350424b37bf178d936e2e71fac5eb82275b)

**Comment by bcmills** [comment]
One more today. This test failure seems potentially relevant to #44343, too.

`greplogs --dashboard -md -l -e 'FAIL: TestParallelSleep' --since=2021-11-05`

[2022-01-27T21:11:21-f5fe5a4/windows-amd64-2016](https://build.golang.org/log/835781b96591576298beadcd03f6886f8b1ec597)

**Comment by bcmills** [comment]
`greplogs --dashboard -md -l -e 'FAIL: TestParallelSleep' --since=2022-01-29`

[2022-03-09T21:31:58-a987aaf/windows-amd64-2016](https://build.golang.org/log/aeae712674d9414d03c2a538d9816bf719c130b3)

**Comment by bcmills** [comment]
Marking as release-blocker for Go 1.19.

(If this isn't a priority to fix for 1.19, we should skip the test so that it doesn't continue to add noise to the builders.)

**Comment by ianlancetaylor** [comment]
I'll replace with a test for the same problem that is not timing dependent.
